src/common/logging/Logging.c

```diff
@@ -196,8 +197,9 @@ char* GetFormattedTime(void)
 {
     time_t rawTime = {0};
     struct tm* timeInfo = NULL;
+    struct tm result = {0};
     time(&rawTime);
-    timeInfo = localtime(&rawTime);
+    timeInfo = localtime_r(&rawTime, &result);
```

Comment:
>localtime_r( [](http://example.com/codeflow?start=15&length=12)

If I am not mistaken, localtime_r does not include tzset (for timezone information) which may need to be separately called (something that localtime does not need)